src/App.js

```diff
@@ -1,11 +1,57 @@
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <h1>My first react app</h1>
-    </div>
-  );
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import Calculator from './components/Calculator';
+import calculate from './Logic/calculate';
```

Comment:
- [ ] calculate.js should not be imported here. Please remove it from here and related code and let's move it to component/calcultor.js. Please have another look at the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/project_math_magicians_events/math-magicians/project_events.md).